ward/_terminal.py

```diff
@@ -301,7 +301,7 @@ def output_dots_module(
 
                 dots_on_line = 0
                 current_path = result.test.path
-                rel_path = str(current_path.relative_to(cwd))
```

Comment:
@JoshKarpel In case you're interested: This was throwing a `ValueError` if you were in a directory deeper than a test you were trying to run, because it couldn't deal with generating a path like `../test_something.py`. Switching to `os.path.relpath` seems to have fixed it (see screenshots in description) :)

I'm still not entirely sure why `relative_to` wasn't working 🤷

Reply:
Huh, I guess this is the documented behavior of `Path.relative_to`. https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to

I guess that the `os.path` version is flexible. TIL!